examples/snowpack/pay.js

```diff
@@ -0,0 +1,23 @@
+import * as Square from '@square/web-sdk';
+
+const applicationId = 'sandbox-sq0idb-lybe_WkfKNAbb3WklswmwA';
+const locationId = 'LKYXSPGPXK05M';
+
+async function start() {
+  const payments = await Square.payments(applicationId, locationId);
```

Comment:
nit: `Square.payments` is not asyc.

Reply:
Keep in mind this is `web-sdk`, the WebPaySDK wrapper, not the underlying SDK itself. But maybe it's a little confusing that the method names are the same, but the signatures are not.

Reply:
there's some disconnect because of how we're handling loading on behalf of the developer. perhaps examples would be clearer using an import like `import { payments } ` but still unclear how much overlap there'll be in audiences.

Reply:
I missed this conversation, and see that we are doing this as `Square.payments` here actually loads the script. I will give some thought into how we can possibly rectify the `web-sdk` package vs the underlying SDK, but this should be fine for right now.